ci-automation/test.sh

```diff
@@ -134,6 +136,12 @@ function test_run() {
     # Make the torcx artifacts available to test implementation
     __prepare_torcx "${arch}" "${vernum}" "${work_dir}"
 
+    # Pass PARALLEL_TESTS to the container
+    if [ -n "${PARALLEL_TESTS}" ] ; then
+        echo "PARALLEL_TESTS=\"${PARALLEL_TESTS}\"" > sdk_container/.env
```

Comment:
Where does this get sourced?

Reply:
It's in the SDK container user's `.bashrc`, which is being generated during SDK container build: https://github.com/flatcar-linux/scripts/blob/main/sdk_lib/Dockerfile.sdk-import#L47-L48